webapp/tests/forms/steps/test_eligibility_steps.py

```diff
@@ -3274,3 +3272,138 @@ def test_if_no_user_b_elster_account_and_no_cheaper_check_then_set_no_info(self)
+                        'investment_income_eligibility': 'yes',
+                        'minimal_investment_income_eligibility': 'no',
+                        'taxed_investment_income_eligibility': 'yes',
+                        'cheaper_check_eligibility': 'no', }
+        with patch('app.forms.steps.eligibility_steps._', MagicMock(side_effect=lambda text_id: text_id)):
+            step = EligibilityMaybeDisplaySteuerlotseStep(
+                endpoint='eligibility',
+                stored_data=session_data,
+                render_info=EligibilityMaybeDisplaySteuerlotseStep.prepare_render_info(
+                    {})
+            )
+            step.handle()
+
+        self.assertEqual(expected_information, step.render_info.additional_info['dependent_notes'])
+
+    def test_if_user_b_has_no_elster_account_and_user_has_minimal_investment_income_check_then_set_correct_info(self):
+        expected_information = [_('form.eligibility.result-note.user_b_elster_account-registration-maybe'),
+                                _('form.eligibility.result-note.capital_investment'),
+                                _('form.eligibility.result-note.deadline')]
+        session_data = {'marital_status_eligibility': 'married',
+                        'separated_since_last_year_eligibility': 'no',
+                        'user_a_has_elster_account_eligibility': 'yes',
+                        'user_b_has_elster_account_eligibility': 'no',
+                        'joint_taxes_eligibility': 'yes',
+                        'alimony_eligibility': 'no',
+                        'pension_eligibility': 'yes',
+                        'investment_income_eligibility': 'yes',
+                        'minimal_investment_income_eligibility': 'yes',
+                        'taxed_investment_income_eligibility': 'yes', }
+        with patch('app.forms.steps.eligibility_steps._', MagicMock(side_effect=lambda text_id: text_id)):
+            step = EligibilityMaybeDisplaySteuerlotseStep(
+                endpoint='eligibility',
+                stored_data=session_data,
+                render_info=EligibilityMaybeDisplaySteuerlotseStep.prepare_render_info(
+                    {})
+            )
+            step.handle()
+
+        self.assertEqual(expected_information, step.render_info.additional_info['dependent_notes'])
+        
+
+    def test_if_no_user_b_elster_account_and_no_cheaper_check_then_set_no_info(self):
+        expected_information = [_('form.eligibility.result-note.deadline')]
+        with patch('app.forms.steps.eligibility_steps._', MagicMock(side_effect=lambda text_id: text_id)):
+            step = EligibilityMaybeDisplaySteuerlotseStep(
+                    endpoint='eligibility',
+                    render_info=EligibilityMaybeDisplaySteuerlotseStep.prepare_render_info(
+                        {}))
+            step.handle()
+
+        self.assertEqual(expected_information, step.render_info.additional_info['dependent_notes'])
```

Comment:
Can we maybe rename the test? As it doesn't test for "no info"? I'd suggest `test_if_no_special_case_then_set_only_deadline_info`